src/main.rs

```diff
@@ -1,28 +1,34 @@
 mod application;
 #[rustfmt::skip]
 mod config;
+mod cli;
 mod login;
 mod session;
 mod utils;
 mod window;
 
+use std::path::PathBuf;
```

Comment:
All this group of use declarations should be grouped below the `use self:...` declarations.